app/assets/stylesheets/article-show.scss

```diff
@@ -351,7 +351,11 @@ article {
       a {
         color: $black;
         padding: calc(0.3vw + 14px);
-        background: darken($lightest-gray, 12%);
+        @include themeable (
```

Comment:
Nice to see the `themeable` mixin in action again 😻. Nice work @Kurtz1993!

Reply:
Thanks @nickytonline 😄